corehq/sql_db/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.db import connections
+
+
+class SqlDbAppConfig(AppConfig):
+    name = 'corehq.sql_db'
+
+    def ready(self):
+        """This gets run when Django starts"""
+        identify_standby_databases()
```

Comment:
This means that if standby machines are switched without restarting Django, Django might tag a db that is not standby as standby. That probably is okay, since that's super unlikely. 

Should we trigger this function in a different way instead of this?

Reply:
I can't think of a scenario where that would happen.